interfaces/builtin/docker_support.go

```diff
@@ -1013,6 +1020,15 @@ func (iface *dockerSupportInterface) AppArmorConnectedPlug(spec *apparmor.Specif
 	if !release.OnClassic {
 		spec.AddSnippet(dockerSupportConnectedPlugAppArmorCore)
 	}
+	// if apparmor supports userns mediation then add this too
+	if apparmor_sandbox.ProbedLevel() != apparmor_sandbox.Unsupported {
+		if features, err := apparmor_sandbox.ParserFeatures(); err == nil {
+			if strutil.ListContains(features, "userns") {
+				spec.AddSnippet(dockerSupportConnectedPlugAppArmorUserNS)
+			}
+		}
+	}
```

Comment:
should we have corresponding unit test changes?

Reply:
Added in f9c75eb661